.vscode/settings.json

```
@@ -1,13 +1,9 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": "explicit",
-    "source.organizeImports.biome": "explicit",
-    "quickfix.biome": "explicit"
+    "quickfix.biome": "explicit",
+    "source.organizeImports.biome": "explicit"
   },
   "editor.defaultFormatter": "biomejs.biome",
```

```suggestion
  "editor.defaultFormatter": "biomejs.biome",
  "[typescript]": {
    "editor.defaultFormatter": "biomejs.biome"
  },
```
Unfortunately, you still need this even if you set it as the global default formatter.

---

orly - been working fine without it for me 🤷‍♂️

---

That depends on local settings, if you have a local default or not